packages/ui-components/src/components/relative-time-picker/test/relative-time-picker.e2e.ts

```diff
@@ -31,7 +31,7 @@ describe('relative-time-picker', () => {
 
 		beforeEach(async () => {
 			page = await newE2EPage();
-			await page.setContent('<kv-relative-time-picker inputWrapperContentVisible="true"></kv-relative-time-picker>');
+			await page.setContent('<kv-relative-time-picker timezoneContentVisible="true" />');
```

Comment:
can you explain this change please?

Reply:
@pedrorebelocosta I did some changes to the name of the props in the relative time picker in one of the last merges but forgot to update the tests. There was no impact big impact in the tests since they have pass before but to keep coherence, I asked Ricardo if he could do this update since the tests failed in this PR. The failure of the test is not related to this change since, as I said, in my PR the tests passed, but this change corrects the name of the props in the tests.

Reply:
alright :D I was out of the loop my bad :)